docs/user/build-notifications.rst

```diff
@@ -17,8 +17,12 @@ Build Status Webhooks:
   Webhooks can be customized through your own template and a variety of variable substitutions.
 
 .. seealso::
-    :doc:`/guides/build-notifications`
-        All the practical steps for setting up build notifications
+    :doc:`/guides/build/email-notifications`
+        Enable email notifications in a second.
```

Comment:
```suggestion
        Enable email notifications on failed builds,
        so you always know that your docs are deploying successfully. 
```

This matches the tone of the other item

Reply:
Agreed! Suggestion was added manually :+1: